src/components/PostInfo/PostInfo.tsx

```diff
@@ -10,37 +10,31 @@ type Props = {
   post: Post,
 };
 
-export const PostInfo: React.FC<Props> = (
-  {
-    post:
-    {
-      title,
-      body,
-      user,
-      comments,
-    },
-  },
-) => (
-  <div className="PostInfo">
-    <div className="PostInfo__header">
-      <h3 className="PostInfo__title">{title}</h3>
-      {user && (
-        <p>
-          {' Posted by  '}
-
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
```

Comment:
```suggestion
        <h3 className="PostInfo__title">
          {title}
        </h3>
        
```
It's a common practice to split into rows and add spaces. It's easier to read and debug such code.